src/gates_function.jl

```diff
@@ -0,0 +1,66 @@
+using ITensors
+include("constants.jl")
+
+# Expected units of the quantities used in gates                                                                    
+# del_m2 = ergs squared
```

Comment:
Similar comment here - indicate the meaning of the inputs to the function. Think of it as a brief "how to" guide for using the function.